superbench/executor/executor.py

```diff
@@ -224,10 +227,10 @@ def exec(self):
                         result = self.__exec_benchmark(context, log_suffix)
                         benchmark_results.append(result)
                 else:
-                    log_suffix = 'micro-benchmark {}'.format(benchmark_name)
+                    log_suffix = benchmark_name
                     logger.info('Executor is going to execute %s.', log_suffix)
                     context = BenchmarkRegistry.create_benchmark_context(
-                        benchmark_name,
+                        benchmark_real_name,
```

Comment:
Is it compatible with previous configuration?

Reply:
yes, there's backward compatibility